packages/plugin-session-replay-browser/src/constants.ts

```diff
@@ -1,5 +1,22 @@
+import { AMPLITUDE_PREFIX } from '@amplitude/analytics-core';
+import { IDBStoreSession } from './typings/session-replay';
+
 export const DEFAULT_EVENT_PROPERTY_PREFIX = '[Amplitude]';
 
 export const DEFAULT_SESSION_REPLAY_PROPERTY = `${DEFAULT_EVENT_PROPERTY_PREFIX} Session Recorded`;
 export const DEFAULT_SESSION_START_EVENT = 'session_start';
 export const DEFAULT_SESSION_END_EVENT = 'session_end';
+
+export const MASK_TEXT_CLASS = 'amp-mask';
+export const UNMASK_TEXT_CLASS = 'amp-unmask';
+export const SESSION_REPLAY_SERVER_URL = 'https://api-secure.amplitude.com/sessions/track';
```

Comment:
I think EU has a different url, this should also be config based.

Reply:
It looks like we use a zone option in the config: 
<img width="776" alt="image" src="https://github.com/amplitude/Amplitude-TypeScript/assets/2287470/90f9f068-bd56-4385-828c-67e0ac1eda9b">
and we toggle the server url (if its not overridden completely) based on that: https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-core/src/config.ts#L86-L86
I'll add a ticket to use this top level config as well.